docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -0,0 +1,192 @@
+
+### `renderNavigationView`
+
+a renderer function for the drawer componenet.
+Provided with a `progress` SharedValue parameter that indicates the progress of drawer opening/closing. Equals `0` when drawer is `closed` and `1` when `open`.
+This can be used by the drawer component to animated its children while the drawer is opening or closing.
+
+### `onDrawerClose`
+
+a function which is called when the drawer is closed.
+
+### `onDrawerOpen`
+
+a function which is called when the drawer is opened.
+
+### `onDrawerSlide`
+
+function. Called when drawer is moving or animating.
+Provided with a `progress` SharedValue parameter that indicates the progress of drawer opening/closing. Equals `0` when drawer is `closed` and `1` when `open`.
+
+### `onDrawerStateChanged`
+
+a function which is called when the status of the drawer changes. It takes two arguments:
+
+- `newState` - interaction state of the drawer. It can be one of the following:
+  - `DrawerState.IDLE`
+  - `DrawerState.DRAGGING`
+  - `DrawerState.SETTLING`
+- `drawerWillShow` - `true` when `drawer` started animating to `open` position, `false` otherwise.
+
+### `enableTrackpadTwoFingerGesture` (iOS only)
+
+enables two-finger gestures on supported devices, for example iPads with trackpads.
+If not enabled the gesture will require click + drag, with enableTrackpadTwoFingerGesture swiping with two fingers will also trigger the gesture.
+
+### `children`
+
+component or function. Children is a component which is rendered in the content view.
+May also be a render function which may accept a `progress` SharedValue as a parameter that indicates the animation progress, similarily to the `renderNavigationView` render function.
+
+### `mouseButton(value: MouseButton)` (Web & Android only)
+
+allows users to choose which mouse button should handler respond to.
+The enum `MouseButton` consists of the following predefined fields:
+
+- `LEFT`
+- `RIGHT`
+- `MIDDLE`
+- `BUTTON_4`
+- `BUTTON_5`
+- `ALL`
+
+Arguments can be combined using `|` operator, e.g. `mouseButton(MouseButton.LEFT | MouseButton.RIGHT)`. Default value is set to `MouseButton.LEFT`.
+
+### `enableContextMenu(value: boolean)` (Web only)
+
+specifies whether context menu should be enabled after clicking on underlying view with right mouse button. Default value is set to `false`.
+
+## Methods
+
+### `openDrawer(options)`
+
+`openDrawer` accepts an optional `options` parameter, which is an object with the following optional properties:
+
+`initialVelocity`: number, the initial velocity of the object attached to the spring. Default `0`.
+`animationSpeed`: number, controls speed of the animation. Default `1`.
+
+### `closeDrawer(options)`
+
+`closeDrawer` accepts an optional `options` parameter, which is an object with the following optional properties:
+
+`initialVelocity`: number, the initial velocity of the object attached to the spring. Default `0`.
+`animationSpeed`: number, controls speed of the animation. Default `1`.
```

Comment:
```suggestion
- `initialVelocity` - initial velocity of the object attached to the spring. Defaults to `0`.
- `animationSpeed`- controls speed of the animation. Defaults to `1`.
```

Reply:
fixed in [060d2da](https://github.com/software-mansion/react-native-gesture-handler/pull/3188/commits/060d2da676256ed4a08d517427bc93327bb76e2d)